api/src/test/java/io/minio/MinioClientTest.java

```diff
@@ -362,6 +363,25 @@ public void testAwsEndpoints()
         "https://s3-accelerate.dualstack.amazonaws.com.cn/mybucket/myobject", url.split("\\?")[0]);
   }
 
+  @Test
```

Comment:
Please remove this obvious test.

Reply:
Why do you prefer not to cover it with a unit test? What if somebody goes to S3Base.java and modifies `close()` method so http client is not closed anymore. Without a unit test it will only be possible to spot by code review